### DISCUSSION TITLE
Strip modified date from APK file

### DISCUSSION THREAD

**Discussion by ghost** [REAndroid/APKEditor#40] [General]
Sometimes distributed APKs comes with date 01-01-1981 01:01 or dates completely stripped out. Can you add an option to strip all modified dates from APK if possible?

**Comment by REAndroid** [comment]
Can you tell me the purpose ?

**Reply by REAndroid** [reply]
Sounds logical, to be more accurate i will record each file time on path-map.json

**Comment by REAndroid** [comment]
Check latest [release](https://github.com/REAndroid/APKEditor/releases/tag/V1.3.4)
